src/bundle/Resources/public/scss/_mixins.scss

```diff
@@ -97,8 +97,9 @@
         outline: none;
 
         .ibexa-icon {
-            fill: $ibexa-color-base-dark;
             transition: fill 0.3s $ibexa-admin-transition;
```

Comment:
This is not yours but maybe` $ibexa-admin-transition-duration` instead `0.3s` :)